src/components/Web3Provider/index.tsx

```diff
@@ -86,7 +86,7 @@ function Updater() {
 }
 
 function trace(event: any) {
-  if (event.action !== 'request') return
```

Comment:
do we not need the prior check as well? this changes the behavior in addition to adding the new guard

Reply:
i guess i'm not sure why we had this check before. @zzmp could you explain?

Reply:
This is a debugging method that is flag-guarded, used for logging JSON-RPC requests to the console.
The conditional used in the guard was selected through observation and duck-typing. If this seems to work, then it's fine to change.